react/finger-exercises/ecmascript/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

Comment:
Remember the [empty line at the end of the file](https://robots.thoughtbot.com/no-newline-at-end-of-file)!  :)